node-commons/src/main/java/org/apache/iotdb/commons/partition/Endpoint.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-package org.apache.iotdb.consensus.common;
+package org.apache.iotdb.commons.partition;
```

Comment:
The Endpoint is not only used in partition module. For instance, the consensus layer also needs to use the Endpoint. So place the Endpoint to another package might be better. Like org.apache.iotdb.commons.tools?

Reply:
Done